en/getting_started/sensor_selection.md

```diff
@@ -40,18 +38,9 @@ They are so important because the autopilot does not have other means to detect
 
 For more information and recommended hardware see: [Airspeed Sensors](../sensor/airspeed.md).
 
-## Tachometer

```

Comment:
It seems support for this was added here: https://github.com/PX4/PX4-Autopilot/pull/14018 and documentation here: https://github.com/PX4/PX4-user_guide/pull/673 I would not completely remove it but rather deprioritize it compared to sensors which are used much more often.